.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug Report
+about: Report a problem using F´
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+|**Field**|**Value**|
+|:---|:---|
+|**_Report Date_**| [DATE] |
+|**_Reported By_**| [USER] |
+|**_Branch/Hash_**| |
```

Comment:
Since most users will be using a fprime release we may want to call this fprime version or release rather than branch.